src/extensions/mvs/load-helpers.ts

```diff
@@ -316,20 +322,22 @@ export function alphaForNode(node: MolstarSubtree<'representation'>): number {
     }
 }
 /** Create value for `colorTheme` prop for `StructureRepresentation3D` transformer from a representation node based on color* nodes in its subtree. */
-export function colorThemeForNode(node: MolstarSubtree<'color' | 'color_from_uri' | 'color_from_source' | 'representation'> | undefined, context: MolstarLoadingContext): StateTransformer.Params<StructureRepresentation3D>['colorTheme'] {
+export function colorThemeForNode(node: MolstarSubtree<'color' | 'color_from_uri' | 'color_from_source' | 'representation'> | undefined, context: MolstarLoadingContext): StateTransformer.Params<StructureRepresentation3D>['colorTheme'] | undefined {
     if (node?.kind === 'representation') {
         const children = getChildren(node).filter(c => c.kind === 'color' || c.kind === 'color_from_uri' || c.kind === 'color_from_source') as MolstarNode<'color' | 'color_from_uri' | 'color_from_source'>[];
         if (children.length === 0) {
-            return {
-                name: 'uniform',
-                params: { value: decodeColor(DefaultColor) },
-            };
+            // If no color child node is specified, use Mol*'s default
+            return undefined;
```

Comment:
Here we are making sure MVS is pretty much Molstar-specific and no one will ever implement it for any other viewer.

Reply:
I disagree. You can still implement explicit colors. And this could be defined as expected behavior (= colors determined by viewer if not specified explicitly).

Reply:
I have an alternate solution to support "vendor extension" to do that. Thanks for the feedback.

Reply:
I thought the original idea was that a MVS view should look roughly the same in all viewers. 
With this, a representation node without specified color can look very different in different viewers.
So as a user, I might think I have created a beautiful view where each chain has a different color, but then I load it in PyMOL it will be all magenta because default per-chain coloring was just Molstar's implementation detail.